capabilities/host.go

```diff
@@ -190,3 +275,44 @@ func (h *Host) GetResource(req GetResourceRequest) ([]byte, error) {
 
 	return responsePayload, nil
 }
+
+// Verify_cert verifies cert's trust against the passed cert_chain, and
+// expiration and validation time of the certificate.
+// Accepts 3 arguments:
+//   - cert: PEM-encoded certificate to verify.
+//   - cert_chain: list of PEM-encoded certs, ordered by trust usage
+//     (intermediates first, root last). If empty, certificate is assumed trusted.
+//   - not_after: string in RFC 3339 time format, to check expiration against.
+//     If None, certificate is assumed never expired.
+func (h *Host) VerifyCert(cert Certificate, certChain []Certificate, notAfter string) (bool, error) {
+	requestObj := CertificateVerificationRequest{
+		Cert:      cert,
+		CertChain: certChain,
+		NotAfter:  notAfter,
+	}
+
+	payload, err := easyjson.Marshal(requestObj)
+	if err != nil {
+		return false, fmt.Errorf("cannot serialize request object: %w", err)
+	}
+
+	// perform callback
+	responsePayload, err := h.Client.HostCall("kubewarden", "crypto", "v1/is_certificate_trusted", payload)
+	if err != nil {
+		return false, err
+	}
+
+	responseObj := CertificateVerificationResponse{}
+	if err := easyjson.Unmarshal(responsePayload, &responseObj); err != nil {
+		return false, fmt.Errorf("cannot unmarshall response object: %w", err)
+	}
+
+	switch responseObj.Trusted {
+	case true:
+		return true, nil
+	case false:
+		return false, fmt.Errorf(responseObj.Reason)
+	default:
+		return false, fmt.Errorf("unknown response from the host")
```

Comment:
`responseObj.Trusted` is a boolean variable, there's no way this switch statement can reach the `default` case.

Can you replace the switch with a simple `if/else` statement?